doc/about.rst

```diff
@@ -586,9 +586,8 @@ Donating to the project
 .......................
 
 If you are interested in donating to the project or to one of our code-sprints,
-you can use the *Paypal* button below or the `NumFOCUS Donations Page
-<https://www.numfocus.org/support-numfocus.html>`_ (if you use the latter,
-please indicate that you are donating for the scikit-learn project).
+please donate via the `NumFOCUS Donations Page
+<https://numfocus.org/donate-to-scikit-learn>`_  (please make sure that you are donating for the scikit-learn project).
```

Comment:
```suggestion
<https://numfocus.org/donate-to-scikit-learn>`_.
```

The page is now scikit-learn specific so I don't think the comment makes sense anymore